src/components/internal/DataViewFilters.jsx

```diff
@@ -24,49 +25,91 @@ const StyledDiv = styled.div`
 	}
 `;
 
-const PopoverDiv = styled.div`
-	&.loading {
-		opacity: .5;
-		pointer-events: none;
-	}
-`
-
 function DataViewFilters(props) {
 	const { t } = useMosaicTranslation();
+	const [state, setState] = useState({
+		anchorEl : null,
+		dropdownOpen : false
+	});
 
-	const activeFilters = props.activeFilters || [];
-	
+	const activeFilters = props.activeFilters.value || [];	
 	const primaryFilters = props.filters.filter(val => val.type === "primary");
 	const primaryFilterNames = primaryFilters.map(val => val.name);
 	const optionalFilters = props.filters.filter(val => val.type !== "primary");
 	
 	const active = optionalFilters.filter(val => activeFilters.includes(val.name));
-	const options = optionalFilters.map(val => ({ label : val.label, value : val.name }));
+	const options = optionalFilters
+    .map((val) => ({ label: val.label, value: val.name }))

```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 4 spaces.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9715290)
```suggestion
		.map((val) => ({ label: val.label, value: val.name }))
```